tools/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_swift_tool_subdirectory(swift-demangle)
 add_swift_tool_subdirectory(lldb-moduleimport-test)
 add_swift_tool_subdirectory(sil-extract)
 add_swift_tool_subdirectory(swift-llvm-opt)
+add_swift_tool_subdirectory(swift-api-digester)
 
 if(SWIFT_HOST_VARIANT STREQUAL "macosx")
   # Only build Darwin-specific tools when deploying to OS X.
```

Comment:
Sorry, I missed this section. I see how my patch could be fixed for just OSX support, but is this something we want for Linux too?